src/Core/Manifest/VersionProvider.php

```diff
@@ -44,7 +44,7 @@ class VersionProvider
      */
     public function getVersion()
     {
-        $key = sprintf('%s-%s', $this->getComposerLockPath(), 'all');
+        $key = preg_replace("/[^A-Za-z0-9]/", '', $this->getComposerLockPath() . 'all');
```

Comment:
Not sure how many people will ever read the output, but we can use a delimiter not to have it all munged together
```suggestion
        $key = preg_replace("/[^A-Za-z0-9]/", '_', $this->getComposerLockPath() . '_all');
```